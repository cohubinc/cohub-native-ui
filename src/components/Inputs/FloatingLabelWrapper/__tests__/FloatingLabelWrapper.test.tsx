import React from "react";
import { mount } from "enzyme";

import FloatingLabelWrapper from "../index";
it("matches snapshot", async () => {
  const wrapper = mount(
    <FloatingLabelWrapper>
      {({ componentProps, setInputRef }) => (
        <input {...componentProps} ref={setInputRef} />
      )}
    </FloatingLabelWrapper>
  );

  expect(wrapper).toMatchSnapshot();
});
