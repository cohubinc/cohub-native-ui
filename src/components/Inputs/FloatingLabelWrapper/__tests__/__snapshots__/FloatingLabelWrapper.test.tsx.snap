// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<FloatingLabelWrapper>
  <div
    className="FloatingLabelWrapper GenericInput "
    style={
      Object {
        "color": "hsl(0, 2%, 12%)",
        "cursor": "text",
        "position": "relative",
      }
    }
  >
    <div
      className="inputWrapper  bd-radius"
    >
      <input
        onBlur={[Function]}
        onClick={[Function]}
        onFocus={[Function]}
        style={
          Object {
            "color": "hsl(0, 2%, 12%)",
            "cursor": "text",
            "textAlign": undefined,
          }
        }
      />
      <span
        className="bar "
      />
    </div>
  </div>
</FloatingLabelWrapper>
`;
